tests/base/develop_pipelines.py

```diff
@@ -86,9 +86,13 @@ def run_model_test(trainer_options, model, on_gpu: bool = True, version=None, wi
             trainer.optimizers, trainer.lr_schedulers, trainer.optimizer_frequencies = \
                 trainer.init_optimizers(pretrained_model)
 
-        # test HPC loading / saving
+        # test HPC saving
         trainer.checkpoint_connector.hpc_save(save_dir, logger)
-        trainer.checkpoint_connector.hpc_load(save_dir, on_gpu=on_gpu)
+        # test HPC loading
+        max_suffix = trainer.checkpoint_connector.max_ckpt_in_folder(save_dir)
```

Comment:
Could this be wrapped into a `get_max_checkpoint_path_from_folder` function ?

Reply:
Refactored, thanks for review!